### ISSUE TITLE
Questions regarding memory stats on out-of-memory warning

### ISSUE THREAD

**Issue by mpodeszwa** [getsentry/sentry-cocoa#363]
Hi,
I received memory warning on user's device with following stats:
```
Model | iPhone 6
Architecture | arm64
Memory | Total: 989.0 MB / Free: 23.3 MB / Usable: 656.4 MB
```
I checked Sentry's code and these values are calculated using:
```
static uint64_t freeMemory(void)
{
    vm_statistics_data_t vmStats;
    vm_size_t pageSize;
    if(VMStats(&vmStats, &pageSize))
    {
        return ((uint64_t)pageSize) * vmStats.free_count;
    }
    return 0;
}

static uint64_t usableMemory(void)
{
    vm_statistics_data_t vmStats;
    vm_size_t pageSize;
    if(VMStats(&vmStats, &pageSize))
    {
        return ((uint64_t)pageSize) * (vmStats.active_count +
                                       vmStats.inactive_count +
                                       vmStats.wire_count +
                                       vmStats.free_count);
    }
    return 0;
}
```

Now my questions. 
- What does usable memory actually mean? :) As far as I'm aware `wire_count` is not usable by applications. 
- Shouldn't `inactive_count` be included in `freeMemory` as well?

**Comment by fetzig** [comment]
tl;dr: good question. no idea.

This part of sentry-cocoa is a hard fork of `KSCrash`.

**Comment by mpodeszwa** [comment]
I don't know why but values seems to be correct. I tested this case on my own device.
Sentry dashboard shows data:
```
Total: 2.0 GB / Free: 35.1 MB / Usable: 1.3 GB
```
And xcode was showing usage by my app ~1.3GB and memory limit 1.42GB. Close enough 😄
